test/test_gehrd.cc

```diff
@@ -38,8 +38,8 @@ void test_gehrd_work( Params& params, bool run )
     params.gflops();
     params.ortho();
 
-    params.error.name( "A - U H U^H\nerror" );
-    params.ortho.name( "I - U U^H\nerror" );
+    params.error.name( "A-UHU^H" );
```

Comment:
There is no space compared with `test/test_geev.cc`

Reply:
The error field is 9 chars wide, so I was trying to keep it within that. But in test_geev, it seemed much harder to read `A'Vl-VlW'` (9 chars) or `A^HVl-VlW^H` (11 chars) without some space, so I adjusted the column width instead. But some headers there I can fit in 9 chars, so I'll fix that.

Reply:
Actually, if rename Vl => U and Vr => V and W => L[ambda], which is what the LAPACK docs* imply anyway, the headers work better in geev:
`A^HU-UL^H` (or `A'U-UL'`, though I shy away from Matlab's ambiguous single-quote notation) and `AV-VL`. I have no idea why LAPACK uses `W` for Lambda.

* zgeev docs:
```
            The right eigenvector v(j) of A satisfies
                             A * v(j) = lambda(j) * v(j)
            where lambda(j) is its eigenvalue.
            The left eigenvector u(j) of A satisfies
                          u(j)**H * A = lambda(j) * u(j)**H
            where u(j)**H denotes the conjugate transpose of u(j).
```

Reply:
I just switched to generic name (error and orth.) to match all other routines.